static/styles/app_components.css

```diff
@@ -65,7 +65,7 @@ input::placeholder {
     /* -- base button styling -- */
     .button {
         padding: 7px 14px;
-        margin: 0;
+        margin: 0.25rem 0;
```

Comment:
You shouldn't modify a general class like `.button` for a specific button. Also, you should use `px` unit as that is what we are using right now. There should be a full stop in your commit message.

Reply:
Ah! got it.

@amanagr I added a new class named button-spacer which solved the issue.

Also, I found another button in Custom emoji section with same issue so I'm adding it in this PR itself.

Kindly review the changes :)